javascript/ql/test/query-tests/LanguageFeatures/SyntaxError/destructingPrivate.js

```diff
@@ -0,0 +1,5 @@
+class C {
+  bar() {
+    {#privDecl} = this;
```

Comment:
add a field declaration for `#privDecl` since this is otherwise an error for other reasons.